src/app/modules/terminal/xterm-attach-addon.ts

```diff
@@ -36,27 +38,7 @@ export class XtermAttachAddon implements ITerminalAddon {
   }
 
   private sendBinary(data: string): void {
-    if (this.socket.readyState !== 1) {
-      return;
-    }
     const buffer = this.encoder.encode(data);
```

Comment:
I think we can move this line to the service as well. In the interest of having as few pre-reqs to using the service method as possible. Keeping the interface simple.